engine/table/src/main/java/io/deephaven/stream/StreamToBlinkTableAdapter.java

```diff
@@ -327,26 +328,44 @@ private TableUpdate doRefresh() {
     @SafeVarargs
     @Override
     public final void accept(@NotNull final WritableChunk<Values>... data) {
+        accept(List.<WritableChunk<Values>[]>of(data));
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     * <p>
+     * Ensures that the blink table sees the full collection of chunks in a single cycle.
+     *
+     * @param datas a collection of per-column {@link WritableChunk chunks} of {@link Values values}. Must all have the
+     *        same {@link WritableChunk#size() size}.
+     */
+    @Override
+    public final void accept(@NotNull Collection<WritableChunk<Values>[]> datas) {
         if (!alive.get()) {
             return;
         }
         // Accumulate data into buffered column sources
         synchronized (this) {
             if (!enqueuedFailures.isEmpty()) {
                 // If we'll never deliver these chunks, dispose of them immediately.
-                SafeCloseable.closeAll(data);
+                for (WritableChunk<Values>[] data : datas) {
+                    SafeCloseable.closeAll(data);
+                }
```

Comment:
This is unfortunate, in that making multiple `closeAll` invocations avoids the guarantee that we'll close *all*.